src/libcollections/vec_deque.rs

```diff
@@ -726,18 +726,18 @@ impl<T> VecDeque<T> {
     /// ```
     /// use std::collections::VecDeque;
     ///
-    /// let mut vector: VecDeque<u32> = VecDeque::new();
+    /// let mut vector = VecDeque::<u32>::new();
```

Comment:
The element type seems to not be important to the example. Even better to leave it out (and let it fall back to i32)?